
00:33:23 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:33:57 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:33:45 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:32:51 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:36:32 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:36:32 - main - WARNING - sending: 196 - cannot write empty image as JPEG
00:36:33 - main - ERROR - <module>: 239 - cannot write empty image as JPEG
Traceback (most recent call last):
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 192, in sending
    bot.send_photo(chat_id=channel_id, photo=page)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/__init__.py", line 1825, in send_photo
    apihelper.send_photo(
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/apihelper.py", line 469, in send_photo
    files = {'photo': util.pil_image_to_file(photo)}
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/service_utils.py", line 62, in pil_image_to_file
    image.convert('RGB').save(photoBuffer, extension, quality=quality)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/PIL/Image.py", line 2432, in save
    save_handler(self, fp, filename)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/PIL/JpegImagePlugin.py", line 637, in _save
    raise ValueError(msg)
ValueError: cannot write empty image as JPEG

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 237, in <module>
    sending()
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 205, in sending
    bot.send_photo(chat_id=channel_id, photo=page)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/__init__.py", line 1825, in send_photo
    apihelper.send_photo(
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/apihelper.py", line 469, in send_photo
    files = {'photo': util.pil_image_to_file(photo)}
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/service_utils.py", line 62, in pil_image_to_file
    image.convert('RGB').save(photoBuffer, extension, quality=quality)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/PIL/Image.py", line 2432, in save
    save_handler(self, fp, filename)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/PIL/JpegImagePlugin.py", line 637, in _save
    raise ValueError(msg)
ValueError: cannot write empty image as JPEG
00:34:40 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:34:44 - main - WARNING - sending: 196 - cannot write empty image as JPEG
00:34:49 - main - ERROR - <module>: 239 - cannot write empty image as JPEG
Traceback (most recent call last):
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 188, in sending
    mediagroup = [telebot.types.InputMediaPhoto(i) for i in mediaphoto_jpeg_skins]
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 188, in <listcomp>
    mediagroup = [telebot.types.InputMediaPhoto(i) for i in mediaphoto_jpeg_skins]
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/types.py", line 6222, in __init__
    media = service_utils.pil_image_to_file(media)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/service_utils.py", line 62, in pil_image_to_file
    image.convert('RGB').save(photoBuffer, extension, quality=quality)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/PIL/Image.py", line 2432, in save
    save_handler(self, fp, filename)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/PIL/JpegImagePlugin.py", line 637, in _save
    raise ValueError(msg)
ValueError: cannot write empty image as JPEG

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 237, in <module>
    sending()
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 201, in sending
    mediagroup = [telebot.types.InputMediaPhoto(i) for i in mediaphoto_jpeg_skins]
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 201, in <listcomp>
    mediagroup = [telebot.types.InputMediaPhoto(i) for i in mediaphoto_jpeg_skins]
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/types.py", line 6222, in __init__
    media = service_utils.pil_image_to_file(media)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/telebot/service_utils.py", line 62, in pil_image_to_file
    image.convert('RGB').save(photoBuffer, extension, quality=quality)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/PIL/Image.py", line 2432, in save
    save_handler(self, fp, filename)
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/PIL/JpegImagePlugin.py", line 637, in _save
    raise ValueError(msg)
ValueError: cannot write empty image as JPEG
14:47:58 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:34:37 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:35:37 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:33:59 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:32:46 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:32:55 - main - WARNING - sending: 197 - cannot write empty image as JPEG
00:32:01 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:32:11 - main - WARNING - sending: 197 - cannot write empty image as JPEG
00:36:20 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:36:28 - main - WARNING - sending: 197 - cannot write empty image as JPEG
00:34:21 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:34:30 - main - WARNING - sending: 197 - cannot write empty image as JPEG
00:34:26 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:34:28 - main - WARNING - sending: 197 - cannot write empty image as JPEG
00:34:57 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:35:05 - main - WARNING - sending: 197 - cannot write empty image as JPEG
00:34:06 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:34:13 - main - WARNING - sending: 197 - cannot write empty image as JPEG
00:37:28 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
